composer/core/logging/logger.py

```diff
@@ -1,15 +1,6 @@
 # Copyright 2021 MosaicML. All Rights Reserved.
 
-"""Base classes, functions, and variables for logger.
-
-Attributes:
```

Comment:
Should we still describe the types upfront? easier than scrolling down into the docs.

Reply:
Sure. I can keep them at the top.